resources/core/google/secrets.ts

```diff
@@ -27,3 +28,13 @@ export const databaseConfigSecretVersion = new gcp.secretmanager.SecretVersion(
   },
   { provider },
 );
+
+export const secretIam = new gcp.secretmanager.SecretIamMember(
```

Comment:
Notice this is an IamMember connected directly to the secret, it is not on project-level. I find this to be best practice. No need for the services to have access to more than they should.